code/datums/greyscale/config_types/greyscale_configs.dm

```diff
@@ -233,7 +233,7 @@
 	name = "Marisa Sneakers"
 	icon_file = 'icons/obj/clothing/shoes.dmi'
 	json_config = 'code/datums/greyscale/json_configs/sneakers_marisa.json'
-	
```

Comment:
If it's not relevant to your PR, you should not touch other files

Reply:
Fixed, thank you